### PR TITLE
Chore: Add organization data tests

### PR DESCRIPTION
**Pull request by razbensimon**
### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)

### Solution
add unit tests

### CODE REVIEW

File: env0/data_organization_test.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	"testing"
+)
+
+var resourceType = "env0_organization"
+var resourceName = "test"
+var resourceFullName = fmt.Sprintf("%s.%s", resourceType, resourceName)
+var organization = client.Organization{
+	Id:   "id0",
+	Name: "name0",
+}
```
Comment (yaronya) [inline env0/data_organization_test.go:16]: @razbensimon so I wouldn't forget or someone will miss it - please avoid using global values. These are global for the package scope 😨

**Comment by razbensimon** [comment]
[UT for env0/data_organization.go](https://trello.com/c/yY6hhFdO/3359-ut-for-env0-dataorganizationgo)

**Comment by razbensimon** [comment]
@yaronya fixed tests as we spoke
also added the `helpers` file as you asked

File: env0/test_helpers.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+// TFSource Enum
+type TFSource string
+
+const (
+	DataResource TFSource = "data"
+	Resource     TFSource = "resource"
+)
+
+func DataSourceAccessor(resourceType string, resourceName string) string {
```
Comment (yaronya) [inline env0/test_helpers.go:15]: Please make sure all functions in this file are private

**Review by yaronya — APPROVED** [review]
🥇

File: env0/test_helpers.go (hunk 2 above)
Comment (razbensimon) [reply env0/test_helpers.go]: fix string newlines + CR 41b1ae8

**Pull request merged**